packages/sdk/src/services/organizations-service.ts

```diff
@@ -0,0 +1,28 @@
+import { OrganizationsManagingOrganizationsApi } from '@meeco/vault-api-sdk';
+import { AuthData } from '../models/auth-data';
+import { EncryptionKey } from '../models/encryption-key';
+import { Environment } from '../models/environment';
+import { vaultAPIFactory } from '../util/api-factory';
+
+/**
+ * Manage organizations from the API.
+ */
+export class OrganizationsService {
+  private api: OrganizationsManagingOrganizationsApi;
+
+  constructor(environment: Environment, vaultAccessToken: string) {
+    this.api = vaultAPIFactory(environment)(vaultAccessToken).OrganizationsManagingOrganizationsApi;
+  }
+
+  public async getLogin(id: string): Promise<AuthData> {
+    const result = await this.api.organizationsIdLoginPost(id);
+    return new AuthData({
+      secret: '',
+      keystore_access_token: '',
+      vault_access_token: result.access_token,
+      data_encryption_key: EncryptionKey.fromRaw(''),
+      key_encryption_key: EncryptionKey.fromRaw(''),
+      passphrase_derived_key: EncryptionKey.fromRaw('')
+    });
```

Comment:
Will these properties eventually be used in an Organisation context?

Reply:
not 100% sure but I think data_encryption_key might be used by the integration engine later. I don't think passphrase_derived_key & key_encryption_key will be used.

Reply:
Okay - as long as it's not likely to just be the auth token in the long run - otherwise re-using the `AuthData` wrapper probably isn't worth it.